### PR TITLE
mark these scripts as bash, they don't work under zsh

### PR DESCRIPTION
**Pull request by ahgittin**

### CODE REVIEW

File: dist/licensing/generate-license-and-notice.sh [hunk 1]
```diff
@@ -146,6 +148,15 @@ if [ ! -z "$LIBRARIES" ] ; then
 fi
 
 process_dependencies() {
+  echo running mvn license-audit-maven-plugin:notices
+  echo mvn -X org.heneveld.maven:license-audit-maven-plugin:notices \
+        -DlicensesPreferred=Apache-2.0,Apache,EPL-1.0,BSD-2-Clause,BSD-3-Clause,CDDL-1.1,CDDL-1.0,CDDL \
+        -DoverridesFile=$TEMP_METADATA_FILE \
+        -DoutputYaml=true \
+        -DoutputFile=$TEMP_NOTICE_DATA_FILE \
+        $@
```
Comment (kemitix) [inline dist/licensing/generate-license-and-notice.sh]: Is this meant to be here, or is it a left over from diagnostic?

Comment (ahgittin) [reply dist/licensing/generate-license-and-notice.sh]: i left it as it's useful ... happy to remove (ideally would be `set +e` or `+x` ? i can never remember the command echo ... but this is only run a few times a year to update the license files so not a big deal either way imo

**Comment by nakomis** [comment]
LGTM; Testing locally and it fixes the issue I was having

No strong feelings on the `echo mvn -X...` comment above. If it were something that was run regularly by a majority of users then I'd say delete it, but for something that's only part of the build process it's fine

**Comment by ahgittin** [comment]
k - merging

**Pull request merged**